docs/src/main/docs/grpc/client/02_configuration.adoc

```diff
@@ -0,0 +1,129 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:javadoc-base-url-api: {javadoc-base-url}?io/helidon/grpc/server
+:pagename: grpc-client-configuration
+:description: Helidon gRPC Client Configuration
+:keywords: helidon, grpc, java, configuration
+
+= gRPC Client Configuration
+
+Configure the gRPC client using the Helidon configuration framework, either programmatically or via a configuration file.
+
+As mentioned earlier, creating a `GRPCServiceClient` involves:
+
+1. Creating a `ClientServiceDescriptor` which describes the methods in the service that this client can invoke.
+2. Creating a gRPC Channel through which the Client communicates with the server.
+
+Both of the above can be configured using the Helidon configuration framework, either programmatically
+or via a configuration file.`
+
+== Configuring the ClientServiceDescriptor
+
+=== Configuring the ClientServiceDescriptor in your code
+
+The easiest way to configure the `ClientServiceDescriptor` is in your application code.
+
+[source,java]
+----
+ClientServiceDescriptor descriptor = ClientServiceDescriptor +
+        .builder(HelloService.class)    // (1)
+        .unary("SayHello")              // (2)
+        .build();                       // (3)
+----
+
+1. Create a  bui;der for a `ClientServiceDescriptor` for the `HelloService`.
+2. Specify that the `HelloService` has a unary method named `SayHello()`. There are many other methods n this class that allow you
+to define `ClientStreaming`, `ServerStreaming` and `Bidirectional Streaming` methods.
+3. Build the `ClientServiceDescriptor`.
+
+=== Configuring the `ClientServiceDescriptor` through a configuration file
```

Comment:
It might be best to take this out. When we finally know what we will be able to configure we can add something back but I suspect that there is nothing we have in configuration that applies specifically to the classes we're talking about here - we don't have any server side configuration for these classes. There are cases on the server where we do configure the `ServiceDescriptor`, for example security, but that is documented in the security section. We cannot actually create a `ServiceDescriptor` from configuration.

Reply:
Removed.